estimatey-api/Estimatey.Application/DeveloperTime/Queries/ListLoggedTimeByDeveloper/ListLoggedTimeByDeveloperQuery.cs

```diff
@@ -58,11 +62,29 @@ public class ListLoggedTimeByDeveloperQueryHandler : IRequestHandler<ListLoggedT
 
     private async Task<List<LoggedTimeDto>> GetLoggedDeveloperTime(ProjectEntity project)
     {
-        var allLoggedTime = await _floatClient.GetLoggedTime(project.FloatId);
-
-        return allLoggedTime
+        var historicLoggedTime = await _dbContext.LoggedTime
+            .Where(_ => !_excludedFloatPersonIds.Contains(_.FloatPerson.FloatId))
+            .Select(_ => new LoggedTimeDto()
```

Comment:
Any reason for not making a `LoggedTimeDto.MapFromEntity` method?

Reply:
Yeah because it was an "interface" dto rather than a response dto didn't feel right for some reason.  No biggie, going to leave like this.